pyatlan/generator/templates/module.jinja2

```diff
@@ -40,7 +40,7 @@ class {{ entity_def.name }}({{super_classes[0]}} {%- if "Asset" in super_classes
     {% include 'referenceable_attributes.jinja2' %}
 {%- else %}
     {% set file_name = 'methods/asset/' + entity_def.name | to_snake_case + '.jinja2' %}
-    {%  if existz('templates/' + file_name) %}
+    {%  if existz(templates_path + '/' + file_name) %}
```

Comment:
Why the change?

Reply:
Related to: https://github.com/atlanhq/atlan-developer-portal/pull/300

I want to run generator scripts from the project root directory, for example: `python3 pyatlan/generator/class_generator.py`. In this case, the hardcoded path in the templates is not respected. Therefore, we should use the `pathlib` path in the templates.